scripts/gen_loader.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/python3
+
+# Copyright (c) 2020 Ben Ashbaugh
```

Comment:
Should be "The Khronos Group"

Reply:
Eek, good catch.  This is the wrong licence header too.  I've fixed both problems.